src/utils/adjust-markup.js

```diff
@@ -12,17 +12,29 @@
 
 const { expand } = require('@emmetio/expand-abbreviation');
 const { setdefault } = require('ferrum');
-const findAndReplace = require('hast-util-find-and-replace');
 const fromDOM = require('hast-util-from-dom');
 const { JSDOM } = require('jsdom');
 const { match: matchUrlBuilder } = require('path-to-regexp');
 const { MarkupConfig } = require('@adobe/helix-shared');
+const visit = require('unist-util-visit');
+const { get } = require('dot-prop');
 const { match } = require('./pattern-compiler');
 const section = require('./section-handler');
+const VDOMTransformer = require('./mdast-to-vdom');
 
-/** Pleceholder variable for the generate template. */
+/** Placeholder variable for the generate template. */
 const PLACEHOLDER_TEMPLATE = /\$\{\d+\}/g;
 
+function findAndReplace(root, insert) {
```

Comment:
@tripodsan my own implementation is almost the same as `"hast-util-find-and-replace"`, but with fewer special cases we don't need, so likely a net positive.